google_fonts/lib/src/generator.dart

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:mustache/mustache.dart';
+
+void main() {
+  final fontsJsonData = readFontsJsonData();
```

Comment:
Add a TODO to integrate with package:build
That would at some point let you avoid checking in the generated files.